app/models/mixins/authentication_mixin.rb

```diff
@@ -79,14 +79,7 @@ def authentication_service_account(type = nil)
   end
 
   def required_credential_fields(type)
-    case type.to_s
-    when "bearer"
-      [:auth_key]
-    when "hawkular"
-      []
-    else
-      [:userid]
-    end
+    []
```

Comment:
@chessbyte 
Authentication connected to a container manager provider use `:auth_key`,
Overriding the `required_credential_fields` method is nice here because it covers all the authentications, :bearer, :hawkular ( and nil, :default ... that should not be used here )

> do we need a similar override for hawkular or the default will work for it?

hawkular will work. All authentications currently connected to a container manager require the [:auth_key] field.